specification/resources/resource-manager/Microsoft.Solutions/stable/2019-07-01/managedapplications.json

```diff
@@ -0,0 +1,2134 @@
+      "properties": {
+        "ApplicationResourceId": {
+          "type": "string",
+          "description": "The parent application id."
+        },
+        "PublisherTenantId": {
+          "type": "string",
+          "description": "The publisher tenant id."
+        },
+        "JitAuthorizationPolicies": {
+          "description": "The JIT authorization policies.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JitAuthorizationPolicies"
+          }
+        },
+        "JitSchedulingPolicy": {          
+          "$ref": "#/definitions/JitRequestProperties",
+          "description": "The JIT request properties."
+        },
+        "ProvisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The JIT request provisioning state."
+        },
+        "JitRequestState": {
+          "readOnly": true,
+          "$ref": "#/definitions/JitRequestState",
+          "description": "The JIT request state."
+        },
+        "CreatedBy": {
+          "$ref": "#/definitions/ApplicationClientDetails",
+          "description": "The client entity that created the JIT request."
+        },
+        "UpdatedBy": {
+          "$ref": "#/definitions/ApplicationClientDetails",
+          "description": "The client entity that last updated the JIT request."
+        }
+      },
+      "required":  [
+        "ApplicationResourceId",
+        "JitAuthorizationPolicies",
+        "JitSchedulingPolicy"
+      ],
+      "description": "Information about JIT request properties"
+    },
+    "JitAuthorizationPolicies": {
+      "properties": {
+        "PrincipalId": {
+          "type": "string",
+          "description": "The the principal id that will be granted JIT access."
+        },
+        "RoleDefinitionId": {
+          "type": "string",
+          "description": "The role definition id that will be granted to the Principal."
+        }
+      },
+      "required": [
+        "PrincipalId",
+        "RoleDefinitionId"
+      ],
+      "description": "The JIT authorization policies."
+    },
+    "JitSchedulingPolicy": {
+      "properties": {
+        "Type": {
+          "$ref": "#/definitions/JitSchedulingType",
+          "description": "The type of JIT schedule."
+        },
+        "Duration": {
+          "type": "string",
+          "format": "duration",
+          "title": "The required duration of the JIT request."
+        },
+        "StartTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The start time of the request."
+        }
+      },
+      "required": [
+        "Type",
+        "Duration",
+        "StartTime"
+      ],
+      "description": "The JIT scheduling policies."
+    },
+    "ApplicationClientDetails": {
+      "properties": {
+        "Oid": {
+          "type": "string",
+          "description": "The client Oid."
+        },
+        "Puid": {
```

Comment:
Aren't PUIDs PII?

Reply:
I've discussed internally, this should be fine.